packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionElement.tsx

```diff
@@ -176,6 +176,8 @@ export default function TransactionElement({ accountAddress, transaction, style,
     const transactionDate = dateFromTimestamp(transaction.time, TimeStampUnit.seconds);
     const transactionTime = withDate ? withDateAndTime(transactionDate) : onlyTime(transactionDate);
     const failed = transaction.status === TransactionStatus.Failed;
+    const isSender = transaction.fromAddress === accountAddress;
+    const amount = isSender && transaction.amount > 0n ? -transaction.amount : transaction.amount;
```

Comment:
Yes, I'm flipping the transaction amount for transactions with a positive `amount` when `fromAddress` matches the currently selected account. As far as I understand, this will never have any effect for transactions returned from the wallet proxy.

Reply:
Wouldn't it make more sense to change the pending transaction's, when they are saved, to ensure that we don't have to do this flip?

Reply:
It seems like it does work currently. I am just averse to this special handling, that relies on an assumption that the other cases (transactions from the wallet proxy) have a certain value? :thinking:

Reply:
When transactions are fetched from the wallet proxy, the sign (+/-) is set based on whether the address used for querying the transactions is sender/receiver. What I'm doing is emulating this behaviour, as users can have both the sending/receiving account of a pending transaction.

Reply:
Yes. Maybe add a TODO that reminds us to check it? :upside_down_face: